### ISSUE TITLE
No work in Edge

### ISSUE THREAD

**Issue by oleteacher** [alyssaxuu/screenity#240]
Love your extension but have not been able to use with Edge. Always use the latest version of Edge.

I install from chrome store, I build from source and the results are the same. Does the extension not work with Edge?

Screenity always reports that I have not allowed cam / mic. I have allowed but the extension does not recognize the fact:

![image](https://github.com/user-attachments/assets/622febc8-78c2-4297-9f62-23578c369e01)

![image](https://github.com/user-attachments/assets/768df324-6439-47a4-9556-0aa92836379e)

**Comment by RussellianReason** [comment]
Exactly the same issue

**Comment by codinginbarn** [comment]
Yes, same issue. Looking the age of this issue, one has to assume the project is in limbo. Hope to see it continue, great extension.

**Comment by alyssaxuu** [comment]
I haven't been focusing on compatibility with Edge, although from testing it myself it seems to work. I wonder if it's OS related, I have tested on macOS - is this happening on Windows / other OS? And is it the case in any website that you attempt recording in? I'll try to test on different devices when I get the chance.

**Comment by oleteacher** [comment]
Windows 10 and/or Windows 11

Not website related, does not work on any website.

As stated, tried in from extension store and also built my own, same issue.

**Comment by oleteacher** [comment]
Sorry, was on a small mobile screen before and fat fingered every key :)

Anyway, have tried on multiple sites and all the same. No errors that I can see in dev.

**Comment by alyssaxuu** [comment]
That's strange. I've just tried it on the latest version of Edge on Windows 10, and Windows 11 as well, and had no issue with the permissions. I'm guessing you've also tried to reset the permissions?

What about when going to an extension page - for example, if you reinstall Screenity stay on the page that automatically opens for setup, and try to open the popup there. Do you see the camera / mic permission settings on the address bar?

<img width="409" alt="Image" src="https://github.com/user-attachments/assets/76935c64-ede6-47b0-abf4-2baa0fbfab0d" />

I've been testing this on a virtual machine via Browserstack so maybe it behaves differently, I'll see if next week I can check it out on an actual Windows laptop. I wonder if it could be some OS-level setting or something along those lines as well.

**Comment by git-0r** [comment]
@alyssaxuu In windows you have to enable camera, mic permissions in settings > Privacy & security > Camera/Microphone.

**Comment by oleteacher** [comment]
@alyssaxuu Thank you for looking into this. Thanks to all for input.

Just to be clear, I had already (before submitting this issue) allowed cam / mic access in security settings. Had to do this for other screen recorders. As shown in my screen snap above, I have allowed access to my cam /mic for the extension. On first use of the extension, I allowed mic / cam.

I had tried on two different computers but of course they were signed into the same Microsoft account with sync on.

Did not want to do this:

![Image](https://github.com/user-attachments/assets/7a20ea3e-a51c-4883-b5b8-e4e3a4e1bceb)

But did. Tried Screenity and it now works.

Beats me. I use other screen recorders (extension based and software app) and never had an issue.

Thank you again Alyssa for your time, sorry to have wasted it :)
